### DISCUSSION TITLE
ligth multitasking

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#91] [Ideas]
Found this. Could be interesting. 
https://hackaday.io/project/196816-antirtos-no-any-rtos-needed-you-will-see

**Comment by VendelinSlezak** [comment]
BleskOS already have some very light multitasking (https://github.com/VendelinSlezak/BleskOS/blob/master/source/drivers/system/scheduler.c) but I will try to write complete interface for multitasking

**Comment by defdefred** [comment]
The ideas of queue of tasks is really nice...
https://github.com/WeSpeakEnglish/ANTIRTOS_MODERN
